libpod/container_commit.go

```diff
@@ -68,6 +68,39 @@ func (c *Container) Commit(ctx context.Context, destImage string, options Contai
 		importBuilder.SetComment(options.Message)
 	}
 
+	// We need to take meta we find in the current container and
+	// add it to the resulting image.
+
+	// Entrypoint - always set this first or cmd will get wiped out
+	importBuilder.SetEntrypoint(c.Spec().Process.Args)
+	// Cmd
+	// We cannot differentiate between cmd and entrypoint here
+	// so we assign args to both
+	importBuilder.SetCmd(c.Spec().Process.Args)
```

Comment:
When we "docker commit" a container there, which of these is/are set in the image's configuration when you then "docker inspect" the result?

Reply:
The both appear to make it